apps/processHandler/libraries/processManager/process.php

```diff
@@ -197,7 +197,8 @@ protected function kill($force = false)
         error_log('pipes closed');
         if (is_resource($this->processDescriptor)) {
             //blocking flow
-            proc_close($this->processDescriptor);
+            //Do not uncomment! it really blocks under certain circumstances, for example when restarting process from gui
+            //proc_close($this->processDescriptor);
```

Comment:
[php.net](https://www.php.net/manual/en/function.proc-close.php) states that
> proc_close() waits for the process to terminate

so proc_close should be moved to the end of this method, but i'm tired of this already